src/components/data-entry/DropdownInput.js

```diff
@@ -41,7 +41,9 @@ class DropdownInput extends Component {
       disabled,
       isRequired,
       error,
-      focused = this.state.focused
+      id,
+      focused = this.state.focused,
+      ...other
```

Comment:
Should we have this here? Should't we just identify all the other possible values instead including aria-label and not pass unknown props down?

Reply:
yeah we def could do that, i mostly draw inspiration from https://github.com/mui-org/material-ui/blob/5daaf9be5115b1cb777fb4cf3d9125baddfe5720/packages/mui-material/src/ButtonBase/ButtonBase.js#L361 that just makes it easier to compose elements, so DropdownInput can act like the antd select. but in this case since the intention is just aria-label, can just be more explicit about it!